catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/security/ShareableMetacardPreIngestPluginTest.java

```diff
@@ -66,25 +66,25 @@ protected Subject getSubject() {
   }
 
   @Test
-  public void testCreateShouldIgnoreNonWorkspaceMetacards() throws Exception {
-    WorkspacePreIngestPlugin wpip = makePlugin(ImmutableMap.of());
+  public void testCreateShouldIgnoreNonShareableMetacards() throws Exception {
+    ShareableMetacardPreIngestPlugin wpip = makePlugin(ImmutableMap.of());
     Metacard metacard = new MetacardImpl();
 
     wpip.process(new CreateRequestImpl(metacard));
     assertThat(metacard.getAttribute(Core.METACARD_OWNER), nullValue());
   }
 
   @Test
-  public void testCreateWorkspaceSuccess() throws Exception {
+  public void testCreateShareableSuccess() throws Exception {
     when(subjectIdentity.getIdentityAttribute()).thenReturn(TEST_ID_ATTR);
     Map<String, SortedSet<String>> attrs =
         ImmutableMap.of(TEST_ID_ATTR, ImmutableSortedSet.of("owner"));
 
-    WorkspacePreIngestPlugin wpip = makePlugin(attrs);
-    WorkspaceMetacardImpl workspace = new WorkspaceMetacardImpl();
+    ShareableMetacardPreIngestPlugin wpip = makePlugin(attrs);
+    ShareableMetacardImpl shareableMetacard = new ShareableMetacardImpl();
```

Comment:
❓ Since technically all metacards should be shareable, we might want to limit the constructors in `ShareableMetacardImpl` to the variants that only take an existing metacard. In that case, it would become more of a `ShareableMetacardDecorator` of sorts. Thoughts?